src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int HOURS_INDEX = 2;
+    private static final int HOUR_SALARY_INDEX = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate dateF = LocalDate.parse(dateFrom, formatter);
+        LocalDate dateT = LocalDate.parse(dateTo, formatter);
```

Comment:
put more effort into naming :) developers spend more time reading code than writing it
```suggestion
        LocalDate dateFrom = LocalDate.parse(dateFrom, formatter);
        LocalDate dateTo = LocalDate.parse(dateTo, formatter);
```